src/evict/evict_lru.c

```diff
@@ -448,7 +451,21 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
         return (0);
 
     __wt_epoch(session, &now);
-    if (WT_TIMEDIFF_SEC(now, cache->stuck_time) > WT_MINUTE * 5) {
+
+    /* Calculate how long the cache has been stuck for and the expiry timeout.*/
+    cache_stuck_duration = WT_TIMEDIFF_SEC(now, cache->stuck_time);
+    cache_stuck_timeout = WT_MINUTE * WT_CACHE_STUCK_TIMEOUT;
+
+#if defined(HAVE_DIAGNOSTIC)
+    /* Enable verbose messaging a short period before the cache is declared stuck
+    for too long and WT crashes. */
+    if ((cache_stuck_duration > (cache_stuck_timeout - WT_CACHE_STUCK_VERBOSE_BEGIN)) &&
+      (!WT_VERBOSE_EVICT_LEVEL_ISSET(session, WT_VERBOSE_DEBUG))) {
```

Comment:
We can just enable verbose for the categories we want directly without checking if already enabled.

Reply:
I included this guard here so that this conditional won't repeatedly be hit once the appropriate verbose level has been set. If we remove it; the verbose level will continually be set every time `evict_server` is called. Is that ok?